src/components/User/Tags/components/TagsListRow.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <li
+    class="app-list__li tags-list-row"
+  >
+    <p
+      class="app-list__cell tags-list-row__name"
+      :title="tag.name"
+    >
+      {{ tag.name }}
+    </p>
+    <div class="tags-list-row__btn-wrp">
+      <button
+        :disabled="isPending"
+        class="app__btn tags-list-row__btn"
+        @click="deleteTag(tag.id)"
+      >
+        {{ "tags-list-row.btn-delete" | globalize }}
+      </button>
+    </div>
+  </li>
+</template>
+
+<script>
+import { api } from '@/api'
+import { ErrorHandler } from '@/utils/ErrorHandler'
+import { TagRecord } from '@/js/records/tag.record'
+
+const EVENTS = {
+  updateList: 'update-list',
+}
+export default {
+  name: 'tags-list-row',
+
+  props: {
+    tag: { type: TagRecord, required: true },
+  },
+
+  data: _ => ({
+    isPending: false,
+  }),
+
+  methods: {
+    async deleteTag (tagId) {
+      this.isPending = true
+      try {
+        await api.deleteWithSignature(`/integrations/marketplace/tags/${tagId}`)
```

Comment:
await api.deleteWithSignature(`/integrations/marketplace/tags/${this.tag.id}`)

Reply:
fixed